files/image_config/monit/monitoring_containers

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/python
```

Comment:
We are migrating SONiC from Python 2 to Python 3. Please change this shebang to `#!/usr/bin/env python3` and make sure your script is Python 3-compliant.

Reply:
Updated! I tested the script with Python 3 on DuTs (single ASIC and multiple ASICs).